Week-10/url-shortener/src/App.js

```diff
@@ -0,0 +1,49 @@
+import "./App.css";
+import { Link, Routes, Route, useLocation } from "react-router-dom";
+import Home from "./pages/Home";
+import Contact from "./pages/Contact";
+import React, { useState } from "react";
+
+function App() {
+  const location = useLocation();
+  let currentPage = "";
+  if (location.pathname === "/Contact") {
```

Comment:
pathname should be in smallcase 
```suggestion
  if (location.pathname === "/contact") {
```